harvester_robot_tests/scripts/node-management/power_off.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Power off node script
+# Usage: power_off.sh &lt;hostname&gt; &lt;host_ip&gt;
+HOSTNAME=$1
+HOST_IP=$2
+echo "Powering off node: $HOSTNAME ($HOST_IP)"
+# AWS example
+if [ "$HOST_PROVIDER" == "aws" ]; then
+INSTANCE_ID=$(cat /tmp/instance_mapping | jq -r ".\"$HOSTNAME\"")
+aws ec2 stop-instances --instance-ids $INSTANCE_ID
+fi
```

Comment:
nit (non-blocking):
Harvester tends to be incompatible with every feasibly deploying on AWS EC2 instances.
The Harvester Cloud project attempted to experiment with AWS but there are too many issues with AWS Harvester Nodes (https://www.github.com/rancher/harvester-cloud/tree/main).
Would it be possible to just remove the entire block?
It also reduces the script's dependencies on external libraries x2 (aws command line tool & jq).
In the current similar script, AWS is not an option, https://github.com/harvester/tests/blob/main/scripts/vagrant/power_off.sh.example